### PR TITLE
LPS-119995 Allow use topic name or id in URLs. LPS-120660 Fix grammar for Topics empty state

### PR DESCRIPTION
**Pull request by LuismiBarcos**

### CODE REVIEW

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js [hunk 1]
```diff
@@ -133,9 +135,9 @@ export default withRouter(({allowCreateTopicInRootTopic, history, section}) => {
 					<NewTopicModal
 						currentSectionId={section && section.id}
 						onClose={() => setVisible(false)}
-						onCreateNavigateTo={(topicName) =>
+						onCreateNavigateTo={(topic) =>
 							historyPushParser(
-								`/questions/${stringToSlug(topicName)}`
+								`/questions/${stringToSlug(topic)}`
```
Comment (javierdearcos) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js:140]: This now could be a topic name or a topic id? How stringToSlug behaves with ids? Could cause some problem?

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js [hunk 2]
```diff
@@ -297,12 +297,16 @@ export default withRouter(
...
+											description={Liferay.Language.get(
+												'there-are-no-questions-inside-this-topic-be-the-first-to-ask-something'
+											)}
 											imgSrc={
 												context.includeContextPath +
 												'/assets/empty_questions_list.png'
 											}
-											title="This topic is empty."
+											title={Liferay.Language.get(
+												'this-topic-is-empty'
```
Comment (javierdearcos) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js:308]: This should not be tthere-are-no-questions-inside-this-topic-be-the-first-to-ask-something? What is the difference?

File: modules/apps/questions/questions-web/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -53,6 +53,7 @@ topic-name=Topic Name
 unmark-as-answer=Unmark As Answer
 update-your-answer=Update Your Answer
 update-your-question=Update Your Question
+use-the-topic-names-in-url=Use the topic names in URL
```
Comment (javierdearcos) [inline modules/apps/questions/questions-web/src/main/resources/content/Language.properties:57]: I doubt about this... Reminds me about the sentence or title discussion. This should not be a title like "Use Topic Names in URL"?

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js (hunk 1 above)
Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js:140]: `stringToSlug` changes the white spaces by `-`. The function receives a string. In the string, we could have a name or an id (that is a string) but it should not be relevant for the function.

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js (hunk 2 above)
Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js:308]: One is the title and the other is the description of the empty state. The image is more descriptive
imagen
For us, each one is a different message inside the empty state.

File: modules/apps/questions/questions-web/src/main/resources/content/Language.properties (hunk 3 above)
Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/content/Language.properties:57]: Oh! Yeah, you are right. My bad, sorry :( 
I Will change it.
Thanks!

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js (hunk 1 above)
Comment (javierdearcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js:140]: Ok

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js (hunk 2 above)
Comment (javierdearcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js:308]: Understood! Sorry for the confusion

**Review by javierdearcos — APPROVED** [review]

**Comment by javierdearcos** [comment]
ci:forward

**Comment by LuismiBarcos** [comment]
ci:forward

**Comment by LuismiBarcos** [comment]
ci:forward

**Pull request closed without merging**